examples/graphics/material-clear-coat.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>PlayCanvas Clear Coat Material</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="icon" type="image/png" href="../playcanvas-favicon.png" />
+    <script src="../../build/output/playcanvas.js"></script>
+    <style>
+        body { 
+            margin: 0;
+            overflow-y: hidden;
+        }
+    </style>
+</head>
+
+<body>
+    <!-- The canvas element -->
+    <canvas id="application-canvas"></canvas>
+
+    <!-- The script -->
+    <script>
+        var canvas = document.getElementById("application-canvas");
+
+        // Create the application and start the update loop
+        var app = new pc.Application(canvas);
+
+        // Set the canvas to fill the window and automatically change resolution to be the same as the canvas size
+        app.setCanvasFillMode(pc.FILLMODE_FILL_WINDOW);
+        app.setCanvasResolution(pc.RESOLUTION_AUTO);
+
+        window.addEventListener("resize", function () {
+            app.resizeCanvas(canvas.width, canvas.height);
+        });
+
+        app.scene.gammaCorrection = pc.GAMMA_SRGB;
+        app.scene.toneMapping = pc.TONEMAP_ACES;
+        // Set the skybox to the 128x128 cubemap mipmap level
+        app.scene.skyboxMip = 1;
+
+        // Create an entity with a camera component
+        var camera = new pc.Entity();
+        camera.addComponent("camera");
+        camera.translate(0, 0, 3);
+        camera.rotate(0, 0, 0);
```

Comment:
Sorry - I keep noticing stuff. This is a no-op so might as well remove.

Reply:
ah - so it is - I'll look out for these in future code.